lib/src/imgproc/imgproc.dart

```diff
@@ -20,6 +21,22 @@ import '../core/termcriteria.dart';
 import '../constants.g.dart';
 import '../opencv.g.dart' as cvg;
 
+List<Vec6f> getTriangles(List<Point> pts) {
+  final p = calloc<ffi.Pointer<cvg.Vec6f>>();
+  final pSize = malloc<ffi.Int>();
+  cvRun(() => CFFI.GetTriangles(pts.cvd.ref, p, pSize));
+  return List.generate(
+      pSize.value,
+      (index) => Vec6f(
+            p.value[index].val1,
+            p.value[index].val2,
+            p.value[index].val3,
+            p.value[index].val4,
+            p.value[index].val5,
+            p.value[index].val6,
+          ));
+}
+
```

Comment:
Here, `p` and `pSize` are neither wrapped in arena nor freed explicitly, so a risk of memory leak exists